src/Sylius/Bundle/ApiBundle/Command/Checkout/ChooseShippingMethod.php

```diff
@@ -13,11 +13,12 @@
 
 namespace Sylius\Bundle\ApiBundle\Command\Checkout;
 
+use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
 use Sylius\Bundle\ApiBundle\Command\OrderTokenValueAwareInterface;
 use Sylius\Bundle\ApiBundle\Command\SubresourceIdAwareInterface;
 
 /** @experimental */
-class ChooseShippingMethod implements OrderTokenValueAwareInterface, SubresourceIdAwareInterface
+class ChooseShippingMethod implements OrderTokenValueAwareInterface, SubresourceIdAwareInterface, IriToIdentifierConversionAwareInterface
```

Comment:
I'm wondering if both SubresourceIdAwareInterface and IriToIdentifierConversionAwareInterface inferfaces shouldn't be extracted to one. Same for most of usage of IriToIdentifierConversionAwareInterface